### PR TITLE
#164798155  Setting Up An Empty React Project

### PR DESCRIPTION
**Pull request by felixkiryowa**
#### What does this PR do?

-  Setting up empty react project

#### Description of Task to be completed?
-  When the Application is run, it should be able to open up a default  page in the browser

#### How should this be manually tested?

- Fetch this branch `git fetch origin ch-setup-react-empty-react-project-164798155`
- Checkout to the branch `git checkout ch-setup-react-empty-react-project-164798155 `
- Run `npm install`
- Run `npm start`
#### What are the relevant pivotal tracker stories?
[#164798155](https://www.pivotaltracker.com/story/show/164798155)

#### Screen shots

Screenshot 2019-04-29 at 15 15 59

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react';
```
Comment (dannylwe) [inline src/components/App.js]: kindly change {component} to { component } to follow convention

**Review by dannylwe — CHANGES_REQUESTED** [review]
Kindly add a new line at the end of files

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+class App extends Component {
+    render() {
```
Comment (adesege) [inline src/components/App.js]: Set up eslint to lint the project. It's part of the requirements. At Andela, we use Airbnb style guide.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (adesege) [inline src/index.html]: Change the title to reflect the name of the Project

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App';
+
+ReactDOM.render(<App />, document.getElementById("app"));
```
Comment (adesege) [inline src/index.js]: Insert a final newline here and in the rest of the files. You can also enable this option in VSCode by adding `"files.insertFinalNewline": true` to your settings.json file

File: README.md [hunk 5]
```diff
@@ -1 +1 @@
-Author's Haven
\ No newline at end of file
+  ### Author's Haven
```
Comment (adesege) [inline README.md]: Add minimal instructions on how to clone and set up the project locally.

**Review by adesege — CHANGES_REQUESTED** [review]
Good job. Please review.

File: src/components/App.js (hunk 2 above)
Comment (felixkiryowa) [reply src/components/App.js]: I have worked on this feedback

File: src/index.html (hunk 3 above)
Comment (felixkiryowa) [reply src/index.html]: I have worked on this feedback

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+
+
+module.exports = {
+    entry:'./src/index.js',
```
Comment (adesege) [inline webpack.config.js]: This file is not formatted. It uses 4 tab spaces instead of 2.

**Review by dorothyas — APPROVED** [review]
Good work @felixkiryowa

**Review by adesege — APPROVED** [review]

**Pull request merged**